### ISSUE TITLE
globally installing npm should not be the recommended way for Mac and Linux

### ISSUE THREAD

**Issue by mcollina** [nodejs/nodejs.org#6361]
Most people that use Node.js _will_ have problems if they install Node.js globally, because they would require npm to install libraries globally with `sudo`. This got 1503 upvotes on [StackOverflow](https://stackoverflow.com/questions/16151018/how-to-fix-npm-throwing-error-without-sudo). This is the page on how to fix the npm permission issue: https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally. This is _scary_ for most newbies, and they will have to go through.

Note that npm recommends to use a version manager as the best way to install Node.js: https://docs.npmjs.com/downloading-and-installing-node-js-and-npm. I agree with them, it's the safest and mo way to install Node.js.

The list of things that creates this kind of permission issues for our users:
1. distro-specific packages (deb, rpm, brew, port, ...)
2. dmg file for Mac OS X

I've personally fixed 100+ computers with this problem across my life. Fixing this requires some not-so-nice solution like `sudo chown -R $(whoami) ~/.npm`.

Another technical solution that is not `nvm` can be found.

**Comment by davidfekke** [comment]
I agree with Matteo. There needs to be a default package manager when you install Node.js. Removing it would create a horrible DX.

**Comment by ovflowd** [comment]
cc @nodejs/tsc I'm in agreement here, and in favour of marking Package Manager as the default tab. Please let us know if there's any 👎 with reaction to this comment, otherwise we'll proceed with a PR.

**Comment by tniessen** [comment]
Personally, I consistently use nvm (in dev environments!). On Windows, I am not sure what a good fit is. Does the Windows installer also regularly result in permission issues?

**Comment by ovflowd** [comment]
> Personally, I consistently use nvm (in dev environments!). On Windows, I am not sure what a good fit is. Does the Windows installer also regularly result in permission issues?

FYI this issue is about making the package managers (ie NVM) the default tab... (I might have misunderstood your comment)

**Comment by tniessen** [comment]
Sorry, I am a bit confused. Is this about the website redesign or about the current production website?

**Comment by ovflowd** [comment]
> Sorry, I am a bit confused. Is this about the website redesign or about the current production website?

Redesign!

**Comment by ovflowd** [comment]
(On the last TSC meeting, @mcollina raised the concern of package managers should be the default way instead of using installers, and then Ive asked him to open an issue after the meeting)

**Comment by richardlau** [comment]
> Personally, I consistently use nvm (in dev environments!). On Windows, I am not sure what a good fit is. Does the Windows installer also regularly result in permission issues?

I haven't used it in a while but it should not do as on Windows `npm install -g` goes into [`AppData\Roaming`](https://github.com/nodejs/node/blob/7f0b80525acec4ef582ab59b3cbce4e82338c93c/tools/msvs/npm/npmrc#L1) (e.g. `\Users\{username}\AppData\Roaming\npm`).

**Comment by mcollina** [comment]
Windows installer is ok.
